server/controllers/authentication.js

```diff
@@ -67,18 +66,27 @@ router.post('/auth/signup', (req, res) => {
 		} else {
 			newUser ['isAdmin'] = true;
 		}
-		// console.log(req.body)
 
 		//This pushes the newly gotten value to the data
-		// users = JSON.stringify(usobj);
 		let usObj = users
 		usObj.push(newUser);
-		// console.log(usObj);
 		const getUser = usObj.find(usr => usr.id === Number(usObj.length));
 		delete getUser['password'];
```

Comment:
is there a reason to delete the user's password??

Reply:
i will fix the delete thing